apps/opik-frontend/src/components/pages-shared/traces/MetricDateRangeSelect/utils.ts

```diff
@@ -1,6 +1,9 @@
 import { INTERVAL_TYPE } from "@/api/projects/useProjectMetric";
 import { DateRangeValue } from "@/components/shared/DateRangeSelect";
 import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+
+dayjs.extend(utc);
```

Comment:
I don't think you should extend `utc` here as it should be done only once (and it's dont in  `opik/apps/opik-frontend/src/lib/date.ts`)